clang/runtime/dpct-rt/include/dpct/dpct.hpp

```diff
@@ -18,8 +18,8 @@ template <class... Args> class dpct_kernel_name;
 template <int Arg> class dpct_kernel_scalar;
 
 #include "atomic.hpp"
+#include "bindless_images.hpp"
```

Comment:
Please keep image.hpp as the top level header file, and bindless_images.hpp as second  level.

Reply:
fixed